kong/db/schema/entities/ca_certificates.lua

```diff
@@ -15,6 +15,7 @@ return {
   fields = {
     { id = typedefs.uuid, },
     { created_at = typedefs.auto_timestamp_s },
+    { updated_at = typedefs.auto_timestamp_s },
```

Comment:
Was there discussion that should it be `typedefs.auto_timestamp_s` or `typedefs.auto_timestamp_ms`?

Reply:
I don't think so. We should keep both (created_at/updated_at) consistent. WDYT?